Source/santasyncservice/SNTSyncStage.mm

```diff
@@ -202,7 +202,9 @@ - (NSData *)dataFromRequest:(NSURLRequest *)request
       errStr = [NSHTTPURLResponse localizedStringForStatusCode:response.statusCode];
     } else {
       code = (long)requestError.code;
+      if (code == 0) code = response.StatusCode;
       errStr = requestError.localizedDescription;
+      if (!errStr) errStr = @"Unknown Error";
```

Comment:
If we are setting `response.StatusCode`, why not also get its `[NSHTTPURLResponse localizedStringForStatusCode:response.statusCode]`